data.sql

```diff
@@ -13,3 +13,24 @@ INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Boarmon', '2000-06-07', 20.4, true, 7);
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Blossom', '1998-10-13', 17, true, 3);
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Ditto', '2022-05-14', 22, true, 4);
+
+/* Insert data into owners tables */
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/206558979-48c6af54-c098-4f26-9801-552d6855d7e0.PNG)

- [ ] You are not following the latest version of readme file. Kindly use the latest version of readme file using these [guidlines](https://github.com/microverseinc/readme-template/blob/master/README.md) and fill the missing things in your readme file 😊

Reply:
Hi @HAMMAS-SALEEM 👋 
I'm using the README template specified for this [project](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md). It is needed to change de README file after all?
Thanks!

P.S.: I face the same situation in the last [approved PR](https://github.com/Camilovelag/Vet-clinic-database/pull/2) 😵‍💫

Reply:
Hey @Camilovelag,
You have to basically use the new readme template given [here](https://github.com/microverseinc/readme-template#readme). If you could update in the next 5-10 mins then I will approve else again ask for changes.

Reply:
Hey @Camilovelag i hope you are doing well.

I don't know why they specified the old readme template but it had been objectively said to us that the latest version of readme will be used in the upcoming projects 😊 

For further you can ask for a second opinion where microverse team will see this matter and they would do fair or if this is Microverse fault then they would update their materials 👍 

You can create an issue in that activity about this thing 😊

Reply:
Hey @Camilovelag I hope you are doing well. I can see that they have updated the readme file and there is a link to your new readme file in this [project](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md) document 👍

Reply:
Done! 😄